app/domain/authentication/authn_jwt/signing_key/fetch_jwks_uri_signing_key.rb

```diff
@@ -31,11 +31,15 @@ def valid_configuration?
           @valid_configuration = jwks_uri_resource_exists?
         end
 
-        def call
+        def fetch_signing_key
```

Comment:
Is it command class or not?

Reply:
its not

Reply:
i called this function call so the cache will use it. Now when there is a wrapper there is no need to so i returned to original name